src/Pages/Main/Main.js

```diff
@@ -1,9 +1,217 @@
+    this.state = {
+      listTransform: 0,
+      listTasteTransform: 0,
+      listReviewTranfrom: 0,
+      listTransition: '1s ease-in-out',
+      isTasteClass: false,
+      date: new Date(),
+      fixDate,
+      countDate: fixDate - new Date(),
+      recommendTaste: '매콤한맛',
+    };
+  }
+
+  handelTasteToggleClick = () => {
+    this.setState({
+      isTasteClass: !this.state.isTasteClass,
+    });
+  };
+
+  handelTasteToggleClick = () => {
+    this.setState({
+      isTasteClass: !this.state.isTasteClass,
+    });
+  };
+
+  handleNextMove = () => {
+    if (this.state.listTransform > -1920 * 4) {
+      this.setState({
+        listTransform: this.state.listTransform - 1920,
+        listTransition: '1s ease-in-out',
+      });
+    } else {
+      this.setState({
+        listTransform: 0,
+        listTransition: '0s',
+      });
+    }
+  };
+
+  handlePrevMove = () => {
+    if (this.state.listTransform < 0) {
+      this.setState({
+        listTransform: this.state.listTransform + 1920,
+        listTransition: '1s ease-in-out',
+      });
+    } else {
+      this.setState({
+        listTransform: -1920 * 4,
+        listTransition: '0s',
+      });
+    }
+  };
+
+  handleTasteNextMove = () => {
+    if (this.state.listTasteTransform > -1574) {
+      this.setState({
+        listTasteTransform: this.state.listTasteTransform - 787,
+        listTransition: '1s ease-in-out',
+      });
+    } else {
+      this.setState({
+        listTasteTransform: 0,
+        listTransition: '0s',
+      });
+    }
+  };
+
+  handleTastePrevMove = () => {
+    if (this.state.listTasteTransform < 0) {
+      this.setState({
+        listTasteTransform: this.state.listTasteTransform + 787,
+        listTransition: '1s ease-in-out',
+      });
+    } else {
+      this.setState({
+        listTasteTransform: -1574,
+        listTransition: '0s',
+      });
+    }
+  };
+
+  handleReveiwNextMove = () => {
+    if (this.state.listReviewTranfrom > -2160) {
+      this.setState({
+        listReviewTranfrom: this.state.listReviewTranfrom - 720,
+        listTransition: '1s ease-in-out',
+      });
+    } else {
+      this.setState({
+        listReviewTranfrom: 0,
+        listTransition: '0s',
+      });
+    }
+  };
+
+  handleReveiwPrevMove = () => {
+    if (this.state.listReviewTranfrom < 0) {
+      this.setState({
+        listReviewTranfrom: this.state.listReviewTranfrom + 720,
+        listTransition: '1s ease-in-out',
+      });
+    } else {
+      this.setState({
+        listReviewTranfrom: -2160,
+        listTransition: '0s',
+      });
+    }
+  };
+
+  tick() {
+    this.setState({
+      countDate: this.state.fixDate - new Date().getTime(),
+    });
+  }
+
+  handleClick = e => {
+    this.setState({
+      recommendTaste: e.target.innerText,
+      isTasteClass: !this.state.isTasteClass,
+    });
+  };
+
+  componentDidMount() {
+    this.timerID = setInterval(() => this.tick(), 1000);
```

Comment:
```suggestion
    this.timerID = setInterval(this.tick, 1000);
```

인자를 넣을 필요가 없다면 함수 선언으로만 해주세요

Reply:
넵 반영했습니다!